test/operators.js

```diff
@@ -167,11 +159,11 @@ prepareAndRunTest('and() ignores non-function arguments', dir, (t, db, raf) => {
   t.end()
 })
 
-prepareAndRunTest('or() ignores non-function arguments', dir, (t, db, raf) => {
+prepareAndRunTest('where() takes only one arguments', dir, (t, db, raf) => {
   const queryTree = query(
     fromDB(db),
-    or(
-      null,
+    where(
+      equal(helpers.seekType, 'contact', { indexType: 'type', indexAll: true }),
       equal(helpers.seekType, 'post', { indexType: 'type', indexAll: true })
```

Comment:
in this case the second argument is ignored, right?

Reply:
I wonder if it would be better to throw an error

Reply:
Hmm, yeah we can make `where` throw.